src/series/point.js

```diff
@@ -6,7 +6,9 @@ import xyBase from './xyBase';
 export default function() {
 
     var decorate = noop,
-        radius = d3.functor(5);
+        symbol = d3.svg.symbol(),
+        size = d3.functor(5 * 5 * 3.14),
```

Comment:
Shall we go with a default of 64 pixels, rather than directly converting old radius